src/components/about/about.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../utils/variables";
+
+.about {
+    padding: $container-padding;
+    justify-content: space-between;
+
+    .about { 
```

Comment:
Here you don't need to write the about again, you can just use the `&-` to concatenate the name. Something like:

```
.about {
  ...
  &-container {
      ...
  }
}
```

Same for the ones under.